packages/boxel/addon/components/boxel/action-container/index.gts

```diff
@@ -46,24 +47,27 @@ const ActionContainer: TemplateOnlyComponent<Signature> = <template>
       />
     {{/if}}
 
-    {{yield (component ActionContainerSection)}}
+    {{yield (component ActionContainerSection) (component BoxelActionChin class='boxel-action-container__footer')}}
 
-    <BoxelActionChin
-      class="boxel-action-container__footer"
-      @state={{if @isComplete "memorialized" "default"}}
-      data-test-boxel-action-footer
-    >
-      <:memorialized as |m|>
-        <m.ActionButton @disabled={{@disabled}} {{on "click" (optional @onClickButton)}}>
-          {{@completeActionLabel}}
-        </m.ActionButton>
-      </:memorialized>
-      <:default as |a|>
-        <a.ActionButton @disabled={{@disabled}} {{on "click" (optional @onClickButton)}}>
-          {{@incompleteActionLabel}}
-        </a.ActionButton>
-      </:default>
-    </BoxelActionChin>
+    {{#if (or @completeActionLabel  @incompleteActionLabel)}}
+      <BoxelActionChin
+        class="boxel-action-container__footer"
+        @state={{if @isComplete "memorialized" "default"}}
+        data-test-boxel-action-footer
+      >
+        <:memorialized as |m|>
+          <m.ActionButton @disabled={{@disabled}} {{on "click" (optional @onClickButton)}}>
+            {{@completeActionLabel}}
+          </m.ActionButton>
+        </:memorialized>
+        <:default as |a|>
+          <a.ActionButton @disabled={{@disabled}} {{on "click" (optional @onClickButton)}}>
+            {{@incompleteActionLabel}}
+          </a.ActionButton>
+        </:default>
+      </BoxelActionChin>
+    {{/if}}
```

Comment:
I wonder if we can remove this part and the associated arguments altogether. The `ActionContainer` component has no current consumers that use it, and leaving this in runs a risk of having two `ActionChin`s.

Reply:
I tend to agree with you, I'll check if the team agrees with us.

Reply:
+1 rip it out!